### PR TITLE
Fix copyright headers in generated files

### PR DESCRIPTION
**Pull request by shyaamsn**
### What this PR does / why we need it
Fix copyright headers to set year to 2023. Without this make verify fails

### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note
Fix copyright headers to set year to 2023
```

### CODE REVIEW

**Review by miclettej — APPROVED** [review]

**Comment by danniel1205** [comment]
should we update the copyright format to be `xxxx-2023` ?

File: apis/addonconfigs/cni/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
-// Copyright 2022 VMware, Inc. All Rights Reserved.
+// Copyright 2023 VMware, Inc. All Rights Reserved.
```
Comment (danniel1205) [inline apis/addonconfigs/cni/v1alpha1/zz_generated.deepcopy.go:4]: question: this seems updated the year in generated file only, do we need to update the source file as well ?

Comment (shyaamsn) [reply apis/addonconfigs/cni/v1alpha1/zz_generated.deepcopy.go:4]: I think doing an audit of copyright headers in all the files and fixing it should be the right thing to do. That seems like a broader exercise.

The goal of this PR is to unblock the CI task `make verify` which unblocks all the PRs.

Comment (shyaamsn) [reply apis/addonconfigs/cni/v1alpha1/zz_generated.deepcopy.go:4]: I have created this issue: https://github.com/vmware-tanzu/tanzu-framework/issues/4227 to address fixing copyright headers in all source files. 

Resolving this comment now to unblock CI pipelines and developer PRs

**Pull request merged**